class-00/lab-a/starter-code/js/app.js

```diff
@@ -71,9 +71,9 @@ function handleClick(event) {
 }
 
 function showList() {
-  for(var i = 0; i < allProducts.length; i++) {
-    var liEl = document.createElement('li');
-    liEl.textContent = allProducts[i].name + ' has ' + allProducts[i].votes + ' votes in ' + allProducts[i].views + ' views';
+  for(const i = 0; i < allProducts.length; i++) {
```

Comment:
Since the iterator (the 'i' variable) starts at 0 and changes to a different number each time it goes through the loop, you want to use 'let' instead of 'const'.